tests/testthat/helper-queue.R

```diff
@@ -17,8 +17,8 @@ MockQueue <- R6::R6Class(
   )
 )
 
-test_queue <- function() {
-  queue <- hintr:::Queue$new()
+test_queue <- function(workers = 2) {
```

Comment:
It will likely be nicer and faster if you set the default here to be zero workers by default.

Reply:
Thanks, updated that.